src/main/java/net/javadiscord/javabot/data/config/guild/HelpConfig.java

```diff
@@ -49,8 +49,14 @@ public class HelpConfig extends GuildConfigItem {
 			You can disable notifications like this using the `/preferences` command.
 			[Post link](%s)
 			""";
-	
-	
+
+	/**
+	 * The message that is sent in a post to tell users that they
+	 * should use discord's code-formatting, provided the bot detects unformatted code.
+	 * Issued by {@link net.javadiscord.javabot.systems.help.AutoCodeFormatter}
+	 */
+	private String formatHintMessage = "> Please format your code to make it more readable. \n> For java, it should look like this: \n```\u200B`\u200B`\u200B`\u200Bjava\npublic void foo() {\n \n}\u200B`\u200B`\u200B`\u200B```";
```

Comment:
To be honest, I would include another line break after the end of the code and before the closing backticks.